test/wildcardEvents/removeAllListeners.js

```diff
@@ -21,7 +21,7 @@ module.exports= {
         ee.on('test.*', function(){
             counter++;
         });
-
+        debugger;
```

Comment:
Might want to remove this debugger statement. 😁

Reply:
Omg. Done :) It seems we need ESLint for pre-commit here, but I'm too lazy :)